h/panels/footer.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+"""The footer displayed at the bottom of most pages."""
+
+from __future__ import unicode_literals
+
+from pyramid_layout.panel import panel_config
+
+
+@panel_config(name='footer', renderer='h:templates/panels/footer.html.jinja2')
+def footer(context, request):
```

Comment:
👍 if it doesn't have any python code in the panel method, there's no point it being a panel -- just shove it in `templates/includes/`.